### DISCUSSION TITLE
F-Droid/Degooglized Phones?

### DISCUSSION THREAD

**Discussion by ramshackle-heather** [fhdm-dev/scoppy#136] [Q&A] [answered]
Is there any chance of an F-Droid release for "degooglized" Android phones without Play Services?

**Comment by fhdm-dev** [comment, marked as answer]
Sorry but no :(

**Reply by ramshackle-heather** [reply]
Thank you, that makes sense. I wish y'all the best!

**Reply by Padm59** [reply]
I have the app installed on the windows subsystem for android and there is also no way of buying the premium features there. Is there another way of getting them to the pc? Already bought them on the phone but my monitor is way more comfortable to use...

**Comment by rodin-ia** [comment]
Maybe you can implement two versions of the program, the regular and “Scoppy Pro”, which will not have built-in purchases but will be paid? Then purchase installation and use of such program on phones without GSI (Degooglized Phones) will be possible.
